build-config/scripts/onie-mk-iso.sh

```diff
@@ -154,9 +154,8 @@ if [ "$UEFI_ENABLE" = "yes" ] ; then
 
     # Generate UEFI format GRUB image
     mkdir -p $RECOVERY_EFI_BOOT_DIR
-    $GRUB_HOST_BIN_UEFI_DIR/grub-mkimage \
-        --format=x86_64-efi \
-        --directory=$GRUB_HOST_LIB_UEFI_DIR \
+    grub-mkimage \
```

Comment:
Is this an issue with the compiler (gcc 5.3.0)?  Is there a newer version that resolves that issue?